src/main/actions/file.js

```diff
@@ -134,6 +134,44 @@ ipcMain.on('AGANI::window::drop', (e, fileList) => {
   }
 })
 
+ipcMain.on('AGANI::response-file-move-to', (e, { pathname }) => {
+  const win = BrowserWindow.fromWebContents(e.sender)
+  if (pathname !== '') {
+    let newPath = dialog.showSaveDialog(win, {
+      buttonLabel: 'Move or rename',
+      nameFieldLabel: 'Filename:',
+      defaultPath: pathname || '~/Untitled.md'
```

Comment:
The default path maybe the same as the `pathname`, so the `~/Untitled.md` is unnecessary?

```javascript
defaultPath: pathname
```

is OK ?